document/CoreFormat.mdk

```diff
@@ -2112,6 +2109,16 @@ if any of those strings accurately describes the construct identified by this ob
 
 If none of those strings accurately describes the construct, `kind` may contain any value specified by the analysis tool.
 
+### `parentKey` property { #logicalLocation-parentKey }
+
+If the logical location represented by the `logicalLocation` object is a nested logical location,
+then the the `logicalLocation` object shall contain a property named `parentKey` whose value is a string 
```

Comment:
Fixed.

---
In reply to: [64121137](https://github.com/sarif-standard/sarif-spec/pull/213#discussion_r64121137) [](ancestors = 64121137)